<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>License owner string</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtLicensePrefix"/>
      </item>
      <item>
       <widget class="QCheckBox" name="cbLicensePartOfKey">
        <property name="text">
         <string>License owner part of license (prefix)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Group size (1-40):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtGroupSize">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>License key:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txtLicenseKey"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblMessage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbGenerate">
        <property name="text">
         <string>&amp;Generate</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbVerify">
        <property name="text">
         <string>Verify</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit">
        <property name="text">
         <string>&amp;Exit</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+I</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
